conda/make_versions.py

```diff
@@ -4,8 +4,9 @@
 import os
 import sys
 import subprocess
+from pathlib import Path
 
-VERSION_PATH = 'mantidimaging/versions.py'
+VERSION_PATH = str(Path(__file__).parent.parent.absolute()) + '/mantidimaging/versions.py'
```

Comment:
Maybe best to go full pathlib here

`VERSION_PATH = Path(__file__).parent.parent / 'mantidimaging' / 'versions.py'`